src/Controller/Form.php

```diff
@@ -87,10 +90,30 @@ public function loadReceiptView() {
 				nocache_headers();
 				header( 'HTTP/1.1 200 OK' );
 
-				// Show donation processing template
+				// Show donation processing template.
 				if ( isConfirmingDonation() ) {
-					include GIVE_PLUGIN_DIR . 'src/Views/Form/defaultFormDonationProcessing.php';
-					exit();
+					$donationId = getDonationIdFromDonorSession();
+
+					/**
+					 * Fire the action hook.
+					 *
+					 * If developer want to verify payment before showing receipt then use `give_handle_donation_confirm` action hook to verify donation.
+					 * You can use src/Helpers/Session/DonationConfirmation/Frontend.php::getPostedData function to get response from payment gateway (if any).
+					 *
+					 * @since 2.7.0
+					 * @param int $donationId
+					 */
+					do_action( 'give_handle_donation_confirmation', $donationId );
+
+					$donation = new Donation( $donationId );
+
+					// Load payment processing video only if donation is in pending status.
```

Comment:
Video?

Reply:
it is `view`.

Reply:
https://github.com/impress-org/givewp/compare/c652b8612198...1a732f01cf52